src/app/pages/private/LifeListPage/displayLife.tsx

```diff
@@ -0,0 +1,28 @@
+import { useListLivesQuery } from '../../../api';
+import NoItemResult from '../../../components/results/NoItemResult';
+
+const DisplayLife = () => {
+    const { data, loading } = useListLivesQuery();
+
+    if (loading) {
+        // not yet loaded
+        return null;
+    }
+
+    const badam = data?.lives || [];
```

Comment:
When you return `[]` or `{}` in such instructions, every react updates will get a different instance which may break optimizations as shallow comparison will not work properly. The solution would be to use `useMemo` to properly return an empty list. here it's fine as we don't re-use the value for lower components but it could have been.